src/Toolkit.Forms/UtilityNetworkTraceTool/UtilityNetworkTraceTool.cs

```diff
@@ -0,0 +1,766 @@
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="GeoView"/> bindable property.
+        /// </summary>
+        public static readonly BindableProperty GeoViewProperty =
+            BindableProperty.Create(nameof(GeoView), typeof(GeoView), typeof(UtilityNetworkTraceTool), null, propertyChanged: OnGeoViewPropertyChanged);
+
+        /// <summary>
+        /// Gets or sets a <see cref="Symbol"/> that represents a starting point.
+        /// </summary>
+        /// <value>
+        /// A <see cref="Symbol"/> that represents a starting point.
+        /// </value>
+        public Symbol? StartingPointSymbol
+        {
+            get => GetValue(StartingPointSymbolProperty) as Symbol;
+            set => SetValue(StartingPointSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a <see cref="Symbol"/> that represents an aggregated multipoint trace result.
+        /// </summary>
+        /// <value>
+        /// A <see cref="Symbol"/> that represents an aggregated multipoint trace result.
+        /// </value>
+        public Symbol? ResultPointSymbol
+        {
+            get => GetValue(ResultPointSymbolProperty) as Symbol;
+            set => SetValue(ResultPointSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a <see cref="Symbol"/> that represents an aggregated polyline trace result.
+        /// </summary>
+        /// <value>
+        /// A <see cref="Symbol"/> that represents an aggregated polyline trace result.
+        /// </value>
+        public Symbol? ResultLineSymbol
+        {
+            get => GetValue(ResultLineSymbolProperty) as Symbol;
+            set => SetValue(ResultLineSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a <see cref="Symbol"/> that represents an aggregated polygon trace result.
+        /// </summary>
+        /// <value>
+        /// A <see cref="Symbol"/> that represents an aggregated polygon trace result.
+        /// </value>
+        public Symbol? ResultFillSymbol
+        {
+            get => GetValue(ResultFillSymbolProperty) as Symbol;
+            set => SetValue(ResultFillSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the GeoView should automatically zoom to show trace results.
+        /// </summary>
+        public bool AutoZoomToTraceResults
+        {
+            get => (bool)GetValue(AutoZoomToTraceResultsProperty);
+            set => SetValue(AutoZoomToTraceResultsProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the layout size that would be appropriate for the current UI state.
+        /// </summary>
+        /// <remarks>
+        /// On mobile devices, it is often appropriate to minimize the control to show more of the map while adding starting points.
+        /// </remarks>
+        public ElementLayoutSizePreference RequestedLayoutSize
```

Comment:
Do we need this property? It appears it's just used in the sample to change layout based on the current state of the workflow. If so, could this instead be a property indicating the current state instead, and an app can choose to change what it looks like based on what the control is currently doing?